samples/azure-quantum/chemistry/QuantumPhaseEstimation/Molecule.ipynb

```diff
@@ -0,0 +1,1775 @@
+       "                                <td>$\\left|15\\right\\rangle$</td>\r\n",
+       "                                <td>$0.0010 + 0.0000 i$</td>\r\n",
+       "                                \r\n",
+       "                                <td>\r\n",
+       "                                    <progress\r\n",
+       "                                        max=\"100\"\r\n",
+       "                                        value=\"9.999970000089987E-05\"\r\n",
+       "                                        style=\"width: 100%;\"\r\n",
+       "                                    > \r\n",
+       "                                    <td>\r\n",
+       "                                    <p id=\"round-eb5b53c0-c77c-43d1-aa43-acfc62551471\"> \r\n",
+       "                                    <script>\r\n",
+       "                                    var num = 9.999970000089987E-05;\r\n",
+       "                                    num = num.toFixed(4);\r\n",
+       "                                    var num_string = num + \"%\";\r\n",
+       "                                     document.getElementById(\"round-eb5b53c0-c77c-43d1-aa43-acfc62551471\").innerHTML = num_string;\r\n",
+       "                                    </script> </p>\r\n",
+       "                                    </td>\r\n",
+       "                                </td>\r\n",
+       "                            \r\n",
+       "                                \r\n",
+       "                                <td style=\"transform: rotate(8.805312532379594E-14deg);\r\n",
+       "                   text-align: center;\">\r\n",
+       "                                 ↑\r\n",
+       "                                </td>\r\n",
+       "                            \r\n",
+       "                            </tr>\r\n",
+       "                        \r\n",
+       "                        </tbody>\r\n",
+       "                    </table>"
+      ],
+      "text/plain": [
+       "|0⟩\t0 + 0𝑖\n",
+       "|1⟩\t0 + 0𝑖\n",
+       "|2⟩\t0 + 0𝑖\n",
+       "|3⟩\t0.9999985000033751 + -6.060528270960681E-17𝑖\n",
+       "|4⟩\t0 + 0𝑖\n",
+       "|5⟩\t0.0009999985000033681 + -1.0352864408264156E-19𝑖\n",
+       "|6⟩\t0 + 0𝑖\n",
+       "|7⟩\t0 + 0𝑖\n",
+       "|8⟩\t0 + 0𝑖\n",
+       "|9⟩\t0 + 0𝑖\n",
+       "|10⟩\t-0.0009999985000033681 + -1.812240014711819E-19𝑖\n",
+       "|11⟩\t0 + 0𝑖\n",
+       "|12⟩\t0 + 0𝑖\n",
+       "|13⟩\t0 + 0𝑖\n",
+       "|14⟩\t0 + 0𝑖\n",
+       "|15⟩\t0.0009999985000033744 + 1.5368146483511924E-18𝑖"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "prepare_state.simulate(JWEncodedData=encoded_data_h2, theta1=0.001, theta2=-0.001, theta3=0.001, nSamples=1)"
```

Comment:
This may be easier to read with `qsharp.config["dump.basisStateLabelingConvention"] = "bitstring"`. The default of `"littleEndian"` makes a lot of sense for arithmetic applications, but I think it'd be easier to see the structure of the single-excitation initial state as |1100⟩ than as |3⟩.

Reply:
That's a helpful suggestion, thanks!